org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corrections/InvertBooleanSubProcessor.java

```diff
@@ -0,0 +1,511 @@
+/*******************************************************************************
+* Copyright (c) 2019 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
```

Comment:
It's better to keep the original copyright info, and append your action in contributors.  For example, append log "extract invert boolean logic to new file".

Reply:
make sense